src/main/java/org/springframework/data/elasticsearch/repository/ElasticsearchRepository.java

```diff
@@ -27,10 +27,13 @@
  * @author Rizwan Idrees
  * @author Mohsin Husen
  * @author Sascha Woo
+ * @author Murali Chevuri
  */
 @NoRepositoryBean
 public interface ElasticsearchRepository<T, ID> extends ElasticsearchCrudRepository<T, ID> {
 
```

Comment:
please add a Javadoc comment for this method that explicitly states that calling this method might lead to a temporary inconststent state until `ElasticsearchRepository.refrsh()`is called